backend/de.metas.manufacturing/src/main/java/org/eevolution/productioncandidate/service/PPOrderCandidateCreateUpdateRequest.java

```diff
@@ -40,14 +43,16 @@
 import org.adempiere.mm.attributes.AttributeSetInstanceId;
 import org.adempiere.warehouse.WarehouseId;
 import org.eevolution.api.PPOrderCreateRequest;
+import org.eevolution.productioncandidate.model.PPOrderCandidateId;
 
 import javax.annotation.Nullable;
 import java.time.Instant;
 
 @Value
 @JsonDeserialize(builder = PPOrderCreateRequest.PPOrderCreateRequestBuilder.class)
-public class PPOrderCandidateCreateRequest
+public class PPOrderCandidateCreateUpdateRequest
```

Comment:
optional boy scout rule: use `@Jacksonized` instead of JsonDeserialize, also drop the PPOrderCreateRequestBuilder

Reply:
changed/removed.